integrations/slack/src/actions/queryLens.ts

```diff
@@ -13,9 +14,35 @@ import {
 } from '../configuration';
 import { acknowledgeQuery } from '../middlewares';
 import { slackAPI } from '../slack';
-import { PagesBlock, QueryDisplayBlock, ShareTools, decodeSlackEscapeChars, Spacer } from '../ui';
+import { QueryDisplayBlock, ShareTools, decodeSlackEscapeChars, Spacer, SourcesBlock } from '../ui';
 import { getInstallationApiClient, stripBotName, stripMarkdown } from '../utils';
 
+export type RelatedSource = {
+    id: string;
+    sourceUrl: string;
+    page: { path: string; title: string };
```

Comment:
Needs two different types for page and snippet no?

Reply:
Not in this case because they conform to the same interface. This is basically post-type in that it's ready to be rendered by the block as a source. if we want to differentiate the snippets from the pages I think we can add more fields here to have a consistent interface.

Reply:
Updated this to be a little clearer by explictly passing the properties needed from the revisionPage